conf/core/DifferentialConflation.conf

```diff
@@ -1,4 +1,9 @@
 {
```

Comment:
Road snapping is turned off in Differential by default, and the config is set up to do it for roads only.  If for some reason we ever used it with other conflation workflows, then we'd need separate config options sets for each to avoid them stepping on each other.